src/current/v23.1/data-domiciling.md

```diff
@@ -80,90 +135,240 @@ ALTER DATABASE movr ADD REGION "us-west1";
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="super-region"><strong>(Recommended)</strong> Use <code>ADD SUPER REGION</code></button>
+  <button class="filter-button page-level" data-scope="placement-restricted"><strong>(Not recommended)</strong> Use <code>PLACEMENT RESTRICTED</code></button>
+</div>
 
-To use this statement, you must set the `enable_multiregion_placement_policy` [session setting]({% link {{ page.version.version }}/set-vars.md %}) or the `sql.defaults.multiregion_placement_policy.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}):
+<div class="filter-content" markdown="1" data-scope="super-region">
+
+To ensure that data on EU-based users, vehicles, etc. from [`REGIONAL BY ROW` tables]({% link {{ page.version.version }}/regional-tables.md %}#regional-by-row-tables) is stored only on EU-based nodes in the cluster, you can use [Super regions]({% link {{ page.version.version }}/multiregion-overview.md %}#super-regions). This will ensure that [regional tables]({% link {{ page.version.version }}/table-localities.md %}#regional-tables) and [regional by row tables]({% link {{ page.version.version }}/table-localities.md %}#regional-by-row-tables) whose [home regions]({% link {{ page.version.version }}/table-localities.md %}) are in the super region will have all of their [replicas]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-replica) stored _only_ in regions that are members of the super region.
+
+To use this statement, you must set the `enable_super_regions` [session setting]({% link {{ page.version.version }}/set-vars.md %}):
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-SET enable_multiregion_placement_policy=on;
+ALTER ROLE ALL SET enable_super_regions = on;
 ~~~
 
+Next, use the [`ALTER DATABASE ... ADD SUPER REGION`]({% link {{ page.version.version }}/alter-database.md %}#add-super-region) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr ADD SUPER REGION "europe" VALUES "europe-west1";
 ~~~
-SET
+
+You have now created a super region with only one region. The updated replica placement settings should start to apply immediately.
+
+{{site.data.alerts.callout_info}}
+[`ALTER DATABASE ... ADD SUPER REGION`]({% link {{ page.version.version }}/alter-database.md %}#add-super-region) does not affect the replica placement for [global tables]({% link {{ page.version.version }}/global-tables.md %}), which are designed to provide fast, up-to-date reads from all [database regions]({% link {{ page.version.version }}/multiregion-overview.md %}#database-regions).
+{{site.data.alerts.end}}
+
+</div>
+
+<div class="filter-content" markdown="1" data-scope="placement-restricted">
+
+**This method is not recommended due to limitations. It is documented for backwards compatibility. Most users should use `ALTER DATABASE ... ADD SUPER REGION`**
```

Comment:
> **This method is not recommended due to limitations. It is documented for backwards compatibility. Most users should use `ALTER DATABASE ... ADD SUPER REGION`**

Can we make this:

> **This method is not recommended and documented for backwards compatibility. Most users should use `ALTER DATABASE ... ADD SUPER REGION` which provides region survival as well as data placement.**

Reply:
Yes, I updated that language ever so slightly to say

> **This method is not recommended, and is documented for backwards compatibility. Most users should use `ALTER DATABASE ... ADD SUPER REGION` which allows for region survival as well as providing data placement.**